torchgen/model.py

```diff
@@ -656,8 +660,9 @@ def from_yaml(
                     )
                     if (
                         dispatch_key is DispatchKey.CompositeImplicitAutograd
-                        and v == cpp.name(func)
-                    ):
+                        or dispatch_key
+                        is DispatchKey.CompositeImplicitAutogradNestedTensor
```

Comment:
I think we can kill this if you follow feedback from https://github.com/pytorch/pytorch/pull/82754/files#r950515109